### PR TITLE
Multi-repository plugin modification

### PR DESCRIPTION
**Pull request by BaronGreenback**
To work with https://github.com/jellyfin/jellyfin/pull/4513

### CODE REVIEW

File: src/controllers/dashboard/plugins/add/index.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ function populateVersions(packageInfo, page, installedPlugin) {
 
     for (let i = 0; i < packageInfo.versions.length; i++) {
         const version = packageInfo.versions[i];
-        html += '<option value="' + version.version + '">' + version.version + '</option>';
+        html += '<option value="' + version.version + '">' + version.version +'[from repository ' + version.repositoryName + ']</option>';
```
Comment (dkanada) [inline src/controllers/dashboard/plugins/add/index.js:24]:
```suggestion
        html += '<option value="' + version.version + '">' + version.version +'from repository ' + version.repositoryName + '</option>';
```

**Review by Camc314 — COMMENTED** [review]
Shouldn't this be translated?

**Comment by BaronGreenback** [comment]
> Shouldn't this be translated?

Ideally yes, "from repository" should be translated - but i'm afraid i've no idea how to do that.

**Comment by Camc314** [comment]
No worries, I can add a commit translating it.

**Review by MrTimscampi — APPROVED** [review]

**Review by thornbill — APPROVED** [review]

**Pull request merged**